src/plugins/intel_gna/src/gna_graph_compiler.cpp

```diff
@@ -837,11 +837,8 @@ void GNAGraphCompiler::PowerPrimitive(InferenceEngine::CNNLayerPtr layer) {
     uint32_t num_rows_out = num_rows_in;
     uint32_t num_padding = ALIGN(num_rows_in, noOfInputsDivisor) - num_rows_in;
 
-    size_t num_data_bytes_out = InferenceEngine::details::product(begin(outputs->getDims()), end(outputs->getDims())) *
-                                outputs->getPrecision().size();
-
-    size_t num_data_bytes_in = InferenceEngine::details::product(begin(input->getDims()), end(input->getDims())) *
-                               input->getPrecision().size();
+    size_t num_data_bytes_out = num_columns_in * (num_rows_out + num_padding) * outputs->getPrecision().size();
```

Comment:
Suggest to move calculation below to some variables - due the fact there are done in multiple places in context of one method
num_rows_out + num_padding
num_rows_in + num_paddin

Reply:
The coding style in this pull request is consistent with the coding style used for all primitives. The purpose of this PR is to fix bugs, not to refactor the code. There is a plan to remove the unused flag 'input_low_precision', and in my opinion, this would be a good opportunity to also refactor the calculation for padding.